test/UniswapV3Staker.unit.spec.ts

```diff
@@ -36,14 +31,18 @@ describe('UniswapV3Staker.unit', async () => {
   let factory: IUniswapV3Factory
   let nft: INonfungiblePositionManager
   let staker: UniswapV3Staker
+  let pool01: string
+  let pool12: string
```

Comment:
I think if there's no tests with two pools, we can just stick to 1 `pool` var  and always test with tokens[0] and tokens[1] for less noise (lmk if you disagree @omarish )

Reply:
`unstakeToken` was using a pool with token[1], token[2] I didn't dig in enough to figure out exactly why, just assumed it was on purpose

Reply:
leftover messiness mostly probably my fault.